pages/about.js

```diff
@@ -67,38 +68,52 @@ const AboutPage = () => (
         <Text id="about.section4-header" />
       </div>
       <div className="about-page__description-text about-page__partners">
-        <PartnerLogo
-          externalUrl="http://shafa-minsk.by/"
-          imgName="shafa_logo.jpg"
-          altText="Лагатып Кніжнай Шафы"
-        />
-        <PartnerLogo
-          externalUrl="http://minsklingfest.by/"
-          imgName="festyval_mou_logo.jpg"
-          altText="Лагатып Фестываля Моў у Мінску"
-        />
-        <PartnerLogo
-          externalUrl="https://vk.com/massaraksh_minsk"
-          imgName="masaraksh_logo.jpg"
-          altText="Лагатып суполкі Масаракш Мінск"
-        />
-        <PartnerLogo
-          classNames="about-page__partner-logo--smallest"
-          externalUrl="https://ethno.by"
-          imgName="ethno_logo.jpg"
-          altText="Лагатып Этнаўсё"
-        />
-        <PartnerLogo
-          classNames="about-page__partner-logo--small"
-          externalUrl="https://libra-gallery.by"
-          imgName="libra_logo.png"
-          altText="Лагатып галерэі Libra"
-        />
-        <PartnerLogo
-          externalUrl="http://brouka.museum.by/"
-          imgName="museum_brouka_logo.jpg"
-          altText="Лагатып Літаратурнага музея Пятруся Броўкі"
-        />
+        {[
+          {
+            id: 'logo-shafa',
+            img: 'shafa_logo.jpg',
+            url: 'http://shafa-minsk.by/',
+          },
+          {
+            id: 'logo-festyval-mou',
+            img: 'festyval_mou_logo.jpg',
+            url: 'http://minsklingfest.by/',
+          },
+          {
+            id: 'logo-masaraksh',
+            img: 'masaraksh_logo.jpg',
+            url: 'https://vk.com/massaraksh_minsk',
+          },
+          {
+            id: 'logo-ethno',
+            img: 'ethno_logo.jpg',
+            url: 'https://ethno.by',
+            className: 'about-page__partner-logo--smallest',
+          },
+          {
+            id: 'logo-libra',
+            img: 'libra_logo.png',
+            url: 'https://libra-gallery.by',
+            className: 'about-page__partner-logo--small',
+          },
+          {
+            id: 'logo-museum-brouka',
+            img: 'museum_brouka_logo.jpg',
+            url: 'http://brouka.museum.by/',
+          },
+        ].map(({ id, img, url, className }) => (
+          <LocaleContext.Consumer>
```

Comment:
`key={id}`

Reply:
1. better to lift up `LocaleContext`, and render it once
2. but every top level page component already have `lang` in props:
https://github.com/babajka/babajka-frontend/blob/develop/pages/_app.js#L107
https://github.com/babajka/babajka-frontend/blob/develop/pages/readme.md#global-props